changelogs/fragments/8954-keycloak-user-federation-add-referral-parameter.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

New arguments are features, not bugfixes:
```suggestion
minor_changes:
```